### DISCUSSION TITLE
Possibility of creating animations ?

### DISCUSSION THREAD

**Discussion by francoisschwarzentruber** [tableaunoir/tableaunoir#131] [Ideas]
Implement a very simple tiny animation editor. By default, tableaunoir will be in a standard blackboard. But we could imagine another mode where we could record and edit animations.

**Comment by Naereen** [comment]
Do you think it is useful to implement it from within tableaunoir?

I've been using a combination of two tools whenever I want to creation animations:
- peek (or any screen recorder) that exports small animations to GIF ;
- and OBStudio for longer animations, exporting both image and sound to MKV or YouTube videos.
![Peek 03-02-2021 12-58](https://user-images.githubusercontent.com/11994719/106744064-85d0b900-661f-11eb-9cd3-0a37b65565cd.gif)

Both allow a fine control on the area that is being recorded, and are well developed and battle-tested tools.
OBS is a very powerful tool, we'll never be able to do as well as they do.

I'm not sure that a tool from within tableaunoir could be useful, honestly.

**Comment by francoisschwarzentruber** [comment]
I agree! And animations may be done via script... I think an animation editor would be very complicated to learn (more than browsing some documentation to find commands to make animations). So I think it will be subsumed by scripts, no ?
